src/dscanner/analysis/useless_initializer.d

```diff
@@ -361,6 +368,17 @@ public:
 		NotKnown nk = NotKnown.init;
 	}, sac);
 
+	// passes
+	assertAnalyzerWarnings(q{
+    	  	@("NOLINT(useless_initializer)")
+			int a = 0;
```

Comment:
check that an error is actually emitted afterwards as well within the same test string